src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1539085816.inc.php

```diff
@@ -0,0 +1,22 @@
+<h1>Build #1539085816</h1>
+<h2>Date: 2018-10-09</h2>
+<div class="changelog">
+    - #118: Add error message for maintenance mode problems
+</div>
+<?php
+
+TCMSLogChange::AddBackEndMessage(
+    'TABLEEDITOR_CMS_CONFIG_MAINTENANCE_ERROR',
+    "Einstellung für 'Alle Webseiten abschalten' konnte nicht gespeichert werden: [{exceptionMessage}]",
+    TCMSLogChange::GetMessageTypeByName('Fehler'),
```

Comment:
I'm afraid this won't work if updates are executed while using the backend with another language as German. This is a conceptual flaw in the GetMessageTypeByName method - I opened a new issue: chameleon-system/chameleon-system/issues/126

For now the best thing to do (*shudder*) is to hard-code the ID (as it is already done e.g. for the default value of AddFrontEndMessage).

Reply:
ok